src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/HttpGoogleDataprocDAO.scala

```diff
@@ -213,7 +211,16 @@ class HttpGoogleDataprocDAO(appName: String,
       val baseConfig = new GceClusterConfig()
         .setTags(List(networkTag.value).asJava)
 
-      (vpcNetwork, vpcSubnet) match {
+      //Dataproc only allows you to specify a subnet OR a network. Subnets will be preferred if present.
+      //High-security networks specified inside of the project will always take precedence over anything
+      //else. Thus, VPC configuration takes the following precedence:
+      // 1) High-security subnet in the project (if present)
```

Comment:
I'm wondering if these rules always hold true, since the code seems to treat subnets as highest priority. For example if I had the following in leonardo.conf:
- `vpcSubnet`
- `projectVPCNetworkLabel`
I think it would pick the subnet over the high security network, right?

Reply:
yup, i agree. `LeoService.getClusterNetwork` and `getClusterSubnet` both have the behaviour of "lookup project label first, then fall back to leo.conf", and pass that into the `ClusterConfiguration` here, and then _this_ code picks subnet over network every time.
@MatthewBemis - is the comment the intended behaviour, or is the code?

Reply:
I believe the comment was the intended behavior